[2022-02-02 21:49:39,669] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_Zone.dowdata manual__2022-02-02T21:49:36.789968+00:00 [queued]>
[2022-02-02 21:49:39,711] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: Question4_Zone.dowdata manual__2022-02-02T21:49:36.789968+00:00 [queued]>
[2022-02-02 21:49:39,711] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:49:39,712] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-02 21:49:39,712] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 21:49:39,746] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): dowdata> on 2022-02-02 21:49:36.789968+00:00
[2022-02-02 21:49:39,753] {standard_task_runner.py:52} INFO - Started process 2496 to run task
[2022-02-02 21:49:39,761] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'Question4_Zone', 'dowdata', 'manual__2022-02-02T21:49:36.789968+00:00', '--job-id', '377', '--raw', '--subdir', 'DAGS_FOLDER/dag_questions4.py', '--cfg-path', '/tmp/tmpxenwmerb', '--error-file', '/tmp/tmp7csyddb1']
[2022-02-02 21:49:39,766] {standard_task_runner.py:77} INFO - Job 377: Subtask dowdata
[2022-02-02 21:49:39,934] {logging_mixin.py:109} INFO - Running <TaskInstance: Question4_Zone.dowdata manual__2022-02-02T21:49:36.789968+00:00 [running]> on host d574c7c5b085
[2022-02-02 21:49:40,098] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Question4_Zone
AIRFLOW_CTX_TASK_ID=dowdata
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T21:49:36.789968+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-02T21:49:36.789968+00:00
[2022-02-02 21:49:40,099] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 21:49:40,100] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://s3.amazonaws.com/nyc-tlc/misc/taxi+_zone_lookup.csv > /opt/***taxi+_zone_lookup.csv']
[2022-02-02 21:49:40,129] {subprocess.py:85} INFO - Output:
[2022-02-02 21:49:41,410] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 21:49:41,474] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=Question4_Zone, task_id=dowdata, execution_date=20220202T214936, start_date=20220202T214939, end_date=20220202T214941
[2022-02-02 21:49:41,555] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 21:49:41,723] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
